### PR TITLE
Update SKILL.tsv

### PR DESCRIPTION
**Pull request by Sunaries**
Part 1.

### CODE REVIEW

File: SKILL.tsv [hunk 1]
```diff
@@ -4244,77 +4244,77 @@ SKILL_20150714_004243	Number of buff you can receive
...
+SKILL_20150714_004294	$Increases the damage dealt on an enemy with [Rim Blow] by 1% per attribute level.
+SKILL_20150714_004295	$Increases the damage dealt on an enemy with [Umbo Blow] by 1% per attribute level.
+SKILL_20150714_004296	$Increases the damage dealt on an enemy with [Shield Lob] by 1% per attribute level.
+SKILL_20150714_004297	$Increases the damage dealt on an enemy with [Butterfly] by 1% per attribute level.
+SKILL_20150714_004298	$Increases the maximum HP of your character by 5% per attribute level while [Swash Buckling] is active. 
+SKILL_20150714_004299	$Enemies hit by [Umbo Blow] can be affected by stun with a 5% chance per attribute level. 
+SKILL_20150714_004300	$Splash will be added onto [Shield Lob] by 1 per attribute level.
+SKILL_20150714_004301	$Increases Block when equipped with a shield by 5% per attribute level.
+SKILL_20150714_004302	$After equipped with a shield, press the C button to use [Guard].
+SKILL_20150714_004303	$Enemies hit by [Umbo Blow] will fall down, increases the knockback power of by 40 per attribute level.
```
Comment (ttgmichael) [inline SKILL.tsv]: extra "of" there

File: SKILL.tsv [hunk 2]
```diff
@@ -4244,77 +4244,77 @@ SKILL_20150714_004243	Number of buff you can receive
 SKILL_20150714_004244	The duration of benefical buff will stop.
 SKILL_20150714_004245	Decrease Cooltime 20 %
```
Comment (ttgmichael) [inline SKILL.tsv:4245]: Should this also change to Cooldown? Line 4245

File: SKILL.tsv [hunk 3]
```diff
@@ -4244,77 +4244,77 @@ SKILL_20150714_004243	Number of buff you can receive
...
+SKILL_20150714_004263	$You can equip the [Two-handed Sword] weapon type.
+SKILL_20150714_004264	$You can equip the [One-handed Spear] weapon type.
+SKILL_20150714_004265	$You can equip the [Two-handed Spear] weapon type.
+SKILL_20150714_004266	$You can equip the [Bow] weapon type.
+SKILL_20150714_004267	$You can equip the [Crossbow] weapon type.
+SKILL_20150714_004268	$You can equip the [Rod] weapon type.
+SKILL_20150714_004269	$You can equip the [Staff] weapon type.
+SKILL_20150714_004270	$You can equip the [Mace] weapon type.
+SKILL_20150714_004271	$You can equip the [Wand] weapon type.
+SKILL_20150714_004272	$You can equip the [Pistol] type weapon.
```
Comment (Sunaries) [inline SKILL.tsv]: Type weapon -> weapon type

File: SKILL.tsv (hunk 1 above)
Comment (grillo-delmal) [inline SKILL.tsv]: $Enemies hit by [Umbo Blow] will fall down, increases the knockback power **of** by 40 per attribute level.

**Comment by ttgmichael** [comment]
I think it's all good :+1:

**Comment by Sunaries** [comment]
Good to merge.

**Comment by grillo-delmal** [comment]
How about "Weaken Defense" then?

**Comment by Sunaries** [comment]
> How about "Weaken Defense" then?

Perfect. Good to merge. :)

**Comment by imcgames** [comment]
Thank you :smile:

**Pull request merged**